imports/ui/home/ChainStatus.jsx

```diff
@@ -20,7 +20,7 @@ export default class ChainStatus extends React.Component {
             totalNumValidators: 0,
             avgBlockTimeType: "",
             avgVotingPowerType: "",
-            blockTimeText: <T>chainStatus.all</T>,
+            blockTimeText: <T _purify={false} avgBlockTimeWindow={numbro(Meteor.settings.public.averageBlockTimeWindow).format({ thousandSeparated: true })}>chainStatus.averageBlockTimeWindow</T>,
```

Comment:
Originally the title for average block time here would be switched with `All Time`, `1h`, `1 day`. What are you going to achieve?
![image](https://user-images.githubusercontent.com/932636/128308245-05602656-8340-48af-b24a-459842ac5c6c.png)